docusaurusify.sh

```diff
@@ -5,6 +5,8 @@ cp ../CHANGELOG.md docs/
 
 cat slug.txt > docs/README.md
 cat ../README.md >> docs/README.md
+# Replace all links that have .docs/ in top level README as we flatten the dir structure
+sed -i 's/(.\/docs/(.\//g' docs/README.md
```

Comment:
You can also use references like `docs/README.md` which is equivalent to `./docs/README.md`